Sources/Views/PaddingLabel/PaddingLabel.swift

```diff
@@ -18,6 +18,10 @@ open class PaddingLabel: UILabel {
     super.drawText(in: rect.inset(by: contentInset))
   }
   
+  override open func textRect(forBounds bounds: CGRect, limitedToNumberOfLines numberOfLines: Int) -> CGRect {
+    super.textRect(forBounds: bounds.inset(by: contentInset), limitedToNumberOfLines: 0)
```

Comment:
can we pass .zero as a default value for `numberOfLines` instead?

Reply:
Tried. I doesn't work with that. Not sure why.